src/applications/check-in/day-of/pages/CheckIn/index.jsx

```diff
@@ -25,6 +28,51 @@ const CheckIn = props => {
   const { token } = context;
   const dispatch = useDispatch();
 
+  const selectForm = useMemo(makeSelectForm, []);
+  const { data } = useSelector(selectForm);
+  // These will be undefined if pages are skipped in day-of. Default to 'yes'
+  const {
+    demographicsUpToDate = 'yes',
+    emergencyContactUpToDate = 'yes',
+    nextOfKinUpToDate = 'yes',
+  } = data;
+
+  const [demographicsFlagsSent, setDemographicsFlagsSent] = useState(false);
+
+  useEffect(
+    () => {
+      if (!isDayOfDemographicsFlagsEnabled) return;
+      const demographicsData = {
+        uuid: token,
+        demographicsUpToDate: demographicsUpToDate === 'yes',
+        emergencyContactUpToDate: emergencyContactUpToDate === 'yes',
+        nextOfKinUpToDate: nextOfKinUpToDate === 'yes',
+      };
+      try {
+        api.v2.patchDayOfDemographicsData(demographicsData).then(resp => {
+          if (resp.data.error || resp.data.errors) {
+            goToErrorPage();
+          } else if (Object.values(demographicsData).includes(false)) {
+            jumpToPage(URLS.SEE_STAFF);
```

Comment:
If I'm understanding this correctly, we kick the veteran to the see staff page if the call is unsuccessful? We might need to talk about this with a larger group but it seems like we might want to trigger a sentry error/alert and let the veteran continue check-in instead?

Reply:
Marking resolved since we're no longer jumping to the Staff page from the Check-in page